apps/laboratory/src/components/Wagmi/WagmiTests.tsx

```diff
@@ -70,6 +71,13 @@ export function WagmiTests() {
             </Heading>
             <WagmiSendCallsWithPaymasterServiceTest />
           </Box>
+
+          <Box>
+            <Heading size="xs" textTransform="uppercase" pb="2">
```

Comment:
Maybe we should add this case to the `wallet.spec.ts` and `email.spec.ts` just like we do for `w3m-open-hook-button` (custom hook button)

Reply:
we should add a disconnect hook ourselves. Will do on a different PR cause `appkit.disconnect()` is currently not working 

For ethers there's no equivalent